resources/s3-objects.go

```diff
@@ -21,21 +22,41 @@ func (n *S3Nuke) ListObjects() ([]Resource, error) {
 	}
 
 	for _, name := range buckets {
-		params := &s3.ListObjectsInput{
+		params := &s3.ListObjectVersionsInput{
 			Bucket: &name,
 		}
 
-		resp, err := n.Service.ListObjects(params)
-		if err != nil {
-			return nil, err
-		}
+		for {
+			resp, err := n.Service.ListObjectVersions(params)
+			if err != nil {
+				return nil, err
+			}
+
+			for _, out := range resp.Versions {
+				resources = append(resources, &S3Object{
+					svc:       n.Service,
+					bucket:    name,
+					key:       *out.Key,
```

Comment:
Don't you need to add the `versionID` to the [`DeleteObjectInput`][1]?

[1]: https://github.com/rebuy-de/aws-nuke/pull/67/files#diff-14f2e0b0cebab6efa8bc9b13cbeca4bbR67